postgres/organisation_service_test.go

```diff
@@ -190,4 +190,28 @@ func TestOrganisationService(t *testing.T) {
 		assert.Nil(t, err, "Getting users in organisation should not error")
 		assert.Len(t, users, 0, "Users list should be empty")
 	})
+
+	t.Run("ok: add user to organisation and check email", func(t *testing.T) {
+		name := tests.Faker.BS()
+		organisation, err := service.CreateOrganisation(ctx, name, nil)
+		tests.Check(err)
+
+		organisation2, err := service.CreateOrganisation(ctx, tests.Faker.BS(), nil)
+		tests.Check(err)
+
+		// Add user to organisation
+		email, err := core.NewEmailAddress("checkmemberemail@example.com")
+		tests.Check(err)
+		user, err := UserService.CreateUser(ctx, tests.Faker.Name(), email, "nl")
+		tests.Check(err)
```

Comment:
Kan je die `tests.Check` allemaal vervangen door `assert.Nil(t, err)`? Dan is dat minder code die we later moeten refactoren als we die Check functie verwijderen 😛

Reply:
aangepast voor de hele file